library/psa_crypto_core.h

```diff
@@ -208,4 +214,97 @@ psa_status_t psa_copy_key_material_into_slot( psa_key_slot_t *slot,
  */
 psa_status_t mbedtls_to_psa_error( int ret );
 
+/** Get the PSA CTR_DRBG context
+ *
+ * \return A pointer to the PSA CTR_DRBG context
+ */
+mbedtls_ctr_drbg_context *psa_get_ctr_drbg_context( void );
+
+/** Import a key in binary format.
```

Comment:
The prototype of the driver entry points are in separate files:  psa_crypto_rsa.h and psa_crypto_ecp.h. The _internal() functions that dispatch (not only, some more refactoring is needed here) to driver entry points are declared in psa_crypto_core.h. As mentioned in the previous comment those functions are planned to disappear when psa_crypto_driver_wrappers.c is generated. We could get rid of them yet but it would be just more commits and we don't really need that for the time being.